### PR TITLE
Tweaked confusing Ruach phrasing

### PR DESCRIPTION
**Pull request by opusforlife2**
**Typo fix**

## Summary

I had to read the line several times for my brain to be able to parse it. Let's just make it simpler.

### CODE REVIEW

File: data/remnant/remnant 2 side missions.txt [hunk 1]
```diff
@@ -441,7 +441,7 @@ mission "Remnant: Shattered Light 2"
 		conversation
 			`Your arrival in the spaceport finds Taely in deep conversation with an elderly Remnant, empty food trays cluttering the table before them. They both turn when you approach, breaking off their conversation.`
 			`	"Thank you for coming, Captain <last>. I would like to introduce you to Ruach. She was a shipyard engineering prefect for many decades, but now spends most of her time as a teacher."`
-			`	Ruach gives a polite bow. "Taely lays too many compliments on me," she gestures with amusement. "I fulfilled my duty doing the best with that which the Embers blessed me, nothing more, nothing less. But it has been rewarding watching someone take everything I taught them and fly with it to heights I could only dream of."`
+			`	Ruach gives a polite bow. "Taely lays too many compliments on me," she gestures with amusement. "I fulfilled my duty making the best of whatever the Embers graciously blessed me with, nothing more, nothing less. But it has been rewarding watching someone take everything I taught them and fly with it to heights I could only dream of."`
```
Comment (warp-core) [inline data/remnant/remnant 2 side missions.txt:444]: The core change here seems to be "that which" -> "whatever".
Moving "with" to the end of the clause seems like a regression in readability to me.
I'm also not sure why "graciously" is being added here. It, and all the other changes seem extraneous.
The simplest change would be:
```suggestion
			`	Ruach gives a polite bow. "Taely lays too many compliments on me," she gestures with amusement. "I fulfilled my duty doing the best with what the Embers graciously blessed me, nothing more, nothing less. But it has been rewarding watching someone take everything I taught them and fly with it to heights I could only dream of."`
```
Or:
```suggestion
			`	Ruach gives a polite bow. "Taely lays too many compliments on me," she gestures with amusement. "I fulfilled my duty doing the best with whatever the Embers graciously blessed me, nothing more, nothing less. But it has been rewarding watching someone take everything I taught them and fly with it to heights I could only dream of."`
```
But, "that which" is generally considered more formal than "what" or "whatever", so while the literal meaning of the words is unchanged, the characterisation they convey is, in my opinion, harmed.

**Comment by Zitchas** [comment]
While I appreciate that her method of speech is archaic and formal, that is intentional in her character. Her exposure to "common" *is* archaic, and she tends to be excessively formal as something of a self-defense when uncomfortable.

**Comment by warp-core** [comment]
> Her exposure to "common" _is_ archaic, and she tends to be excessively formal as something of a self-defense when uncomfortable.

It seems like she is signing those words, not speaking whatever language is used in human space.

**Comment by Zitchas** [comment]
Yes, I definitely acknowledge that her phrasing is unusual.

**Comment by Zitchas** [comment]
Sorry, but no, not particularly.

**Comment by Saugia** [comment]
I don't believe "With that which was given..." is grammatically incorrect - "using something which was given to me by blank"; it is an archaic way of speaking, yes, but I don't believe it is a grammatical error. I can't find anything after some researching to say that it is an incorrect phrase, if anything, I've found a few cases where it is written that way, and I think it sounds fine too.

In the first place, this is Remnant dialogue, and because it is archaic, I think it's fine if it sounds odd. If you look at a lot of the game's Quarg speak, you might find that to be odd as well. In this particular example, I don't think we need to be this critical of language that is of a population of humans that have long since separated from it.

The PR summary suggests you had to read it a few times to parse it, and that the purpose of the PR is "let's to just make it simpler." I don't think, in this case, this is something that needs to be made simpler. It being oddly-sounding is partly in the intent. Who is let's/us? It seems several others believe it to be appropriate sounding, and if there is no concrete evidence that it is not grammatically incorrect, I don't think this is a necessary change (again, in this particular case).

**Comment by warp-core** [comment]
As far as I can tell, the original text ("doing the best with that which the Embers blessed me,") is grammatically correct.
You mentioned previously that you feel "with" should come after "blessed". I do not believe this is a requirement.


> I got that, which is why I tried to maintain a flowery tone. However, my main gripe here is that the "with" has to come after "blessed me", otherwise it doesn't make sense at all. You're "blessed with" good health, not "blessed" good health.

Your example could be written in the same style as the dialog in question as: "The good health with which the Embers have blessed me."

**Comment by Zireael07** [comment]
English major here: "With that which was given..." is grammatically correct.

"doing the best with that which the Embers blessed me" is clunky at best and quite possibly ungrammatical (I am having trouble parsing that.

I'd rephrase to "doing the best with what the Embers blessed me with", since as opus mentioned the second with is clearly missing

**Comment by warp-core** [comment]
I asked @roadrunner56, @Anarchist2, and @Quantumshark what they thought. They said:
image

**Comment by Zitchas** [comment]
If this was in narration, I would definitely be in favor of changing it. 

Broadly speaking, though, as a character dialog, though, it doesn't need to be proper grammar. I'm not just speaking about Ruach here, at this point. If a character is speaking, they can get tongue tied, they can reuse the same word in the sentence twice, they can get overly formal to the point of being problematic, they can most definitely make grammatical mistakes, etc.

I'll definitely be keeping this in mind; but just fixing their grammar isn't always the solution.
